### PR TITLE
fix: Ensure browser contract tests run during top-level build.

### PR DESCRIPTION
**Pull request by kinyoklion**

### CODE REVIEW

File: packages/sdk/browser/contract-tests/entity/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "description": "Contract test service implementation for @launchdarkly/js-client-sdk",
   "scripts": {
     "start": "vite --open=true",
+    "build": "tsc --noEmit && vite build",
```
Comment (kinyoklion) [inline packages/sdk/browser/contract-tests/entity/package.json:9]: There wasn't a build command, so it didn't run with the root build.

File: packages/sdk/browser/contract-tests/entity/src/ClientEntity.ts
Comment (kinyoklion) [inline packages/sdk/browser/contract-tests/entity/src/ClientEntity.ts:1]: I forgot to update this when I removed AutoEnvAttributes.

File: packages/sdk/browser/contract-tests/entity/package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "type": "module",
   "description": "Contract test service implementation for @launchdarkly/js-client-sdk",
   "scripts": {
-    "start": "vite --open=true",
+    "start": "tsc --noEmit && vite --open=true",
```
Comment (kinyoklion) [inline packages/sdk/browser/contract-tests/entity/package.json:8]: Make sure the typescript it valid when running the dev server.

File: packages/sdk/browser/contract-tests/entity/src/ClientEntity.ts [hunk 3]
```diff
@@ -72,6 +66,8 @@ function makeSdkConfig(options: SDKConfigParams, tag: string) {
     };
   }
 
+  cf.fetchGoals = false;
```
Comment (kinyoklion) [inline packages/sdk/browser/contract-tests/entity/src/ClientEntity.ts:69]: Don't fetch goals. It confuses the test harness.

File: packages/sdk/browser/src/BrowserClient.ts [hunk 4]
```diff
@@ -135,8 +136,8 @@ export class BrowserClient extends LDClientImpl {
     };
   }
 
-  override async identify(context: LDContext): Promise<void> {
-    await super.identify(context);
+  override async identify(context: LDContext, identifyOptions?: LDIdentifyOptions): Promise<void> {
```
Comment (kinyoklion) [inline packages/sdk/browser/src/BrowserClient.ts:139]: I missed this, which doesn't make the SDK not work, but does cause problems for the test harness.

**Review by tanderson-ld — APPROVED** [review]

**Pull request merged**